### ISSUE TITLE
resolution conflict with `photutils` between `jwst` and `drizzlepac`

### ISSUE THREAD

**Issue by zacharyburnett** [spacetelescope/stenv#47]
when building `stenv-dev.yml`:
```shell
conda env create --file stenv-dev.yml
```
```
INFO: pip is looking at multiple versions of jwst to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install drizzlepac==3.4.1rc1.dev70+ga830427 and jwst==1.6.3.dev44+gdc52c6ec because these package versions have conflicting dependencies.

The conflict is caused by:
    jwst 1.6.3.dev44+gdc52c6ec depends on photutils>=1.4.0
    drizzlepac 3.4.1rc1.dev70+ga830427 depends on photutils<1.4.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

Currently, in resolving `stenv-latest.yml` and `stenv-stable.yml`, the version of `drizzlepac` is set to be `3.4.0`:
https://github.com/spacetelescope/stenv/runs/7999380317?check_suite_focus=true#step:12:115

**Comment by zacharyburnett** [comment]
@nden

**Comment by zacharyburnett** [comment]
This conflict is resolved by setting `drizzlepac` to version `3.4.0`:
https://github.com/spacetelescope/drizzlepac/blob/c50e3a8240de90c10426903c8f7b74a7c477aef5/setup.cfg#L50
```ini
    photutils>=1.0.0
```
versus the current `master`:
https://github.com/spacetelescope/drizzlepac/blob/a8304273c41a610552d9752496401872de8fe74f/setup.cfg#L50
```ini
    photutils<1.4.0
```
